addons/mrp_subcontracting/models/stock_picking.py

```diff
@@ -20,6 +20,13 @@ class StockPicking(models.Model):
         [('hide', 'Hide'), ('facultative', 'Facultative'), ('mandatory', 'Mandatory')],
         compute='_compute_display_action_record_components')
 
+    def _compute_mrp_production_ids(self):
+        # OVERRIDE to hide subconstracting MO from a resupply picking
+        super()._compute_mrp_production_ids
+        for picking in self:
```

Comment:
There is no place we want to see the subcontracting production orders. The fix could be simpler by hiding any MO having their picking_type_id archived like the subcontracting one (and thus the patch could be done in mrp module)

Reply:
Thanks for the suggestion, I updated the PR